assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -82,15 +84,19 @@ function DashboardIdeasWidget( props ) {
 	const { defaultActiveTabIndex, Widget, WidgetReportError } = props;
 
 	const [ trackedWidgetView, setTrackedWidgetView ] = useState( false );
-
-	const newIdeas = useSelect( ( select ) =>
-		select( MODULES_IDEA_HUB ).getNewIdeas()
+	const [ triggeredSurvey, setTriggeredSurvey ] = useState( false );
+
+	const { newIdeas, savedIdeas, draftIdeas, interactionsCount } = useSelect(
+		( select ) => ( {
+			newIdeas: select( MODULES_IDEA_HUB ).getNewIdeas(),
+			savedIdeas: select( MODULES_IDEA_HUB ).getSavedIdeas(),
+			draftIdeas: select( MODULES_IDEA_HUB ).getDraftPostIdeas(),
+			interactionsCount: select( MODULES_IDEA_HUB ).getInteractionCount(),
```

Comment:
Let's make this singular like it is in the selector (and PHP).

```suggestion
			interactionCount: select( MODULES_IDEA_HUB ).getInteractionCount(),
```